client/jetpack-cloud/sections/partner-portal/invoices-list/index.tsx

```diff
@@ -0,0 +1,133 @@
+import { Button, Card, Gridicon } from '@automattic/components';
+import formatCurrency from '@automattic/format-currency';
+import { useTranslate } from 'i18n-calypso';
+import { ReactElement } from 'react';
+import Badge from 'calypso/components/badge';
+import FormattedDate from 'calypso/components/formatted-date';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+import TextPlaceholder from 'calypso/jetpack-cloud/sections/partner-portal/text-placeholder';
+import useInvoicesQuery from 'calypso/state/partner-portal/invoices/hooks/use-invoices-query';
+import type { Invoice } from 'calypso/state/partner-portal/types';
+import './style.scss';
+
+// Component extracted for readability and not meant to be reusable as the header and card grid (.invoice-list__row)
+// have to be identical for the faux table columns to align visually.
+function InvoiceCard( { dueDate, status, total, pdfUrl }: Invoice ): ReactElement {
+	const translate = useTranslate();
+	const moment = useLocalizedMoment();
+	const dueDateMoment = moment( dueDate );
+
+	let badgeType = 'info';
+	let badgeLabel = translate( 'Draft' );
+
+	switch ( status ) {
+		case 'open':
+			badgeType = 'info-blue';
+			badgeLabel = translate( 'Open' );
+
+			if ( dueDateMoment.isBefore( moment() ) ) {
+				badgeType = 'warning';
+				badgeLabel = translate( 'Past due' );
+			}
+			break;
+
+		case 'paid':
+			badgeType = 'success';
+			badgeLabel = translate( 'Paid' );
+			break;
+
+		case 'uncollectible':
+			badgeType = 'error';
+			badgeLabel = translate( 'Uncollectible' );
+			break;
+
+		case 'void':
+			badgeType = 'info';
+			badgeLabel = translate( 'Void' );
+			break;
+	}
+
+	return (
+		<Card compact>
+			<div className="invoices-list__row">
+				<div className="invoices-list__due-date">
+					<FormattedDate date={ moment( dueDate ) } format="ll" />
+				</div>
+
+				<div className="invoices-list__status">
+					<Badge type={ badgeType }>{ badgeLabel }</Badge>
+				</div>
+
+				<div className="invoices-list__total">{ formatCurrency( total, 'USD' ) }</div>
+
+				<div className="invoices-list__download">
+					{ pdfUrl && (
+						<Button compact href={ pdfUrl } download>
+							{ translate( 'Download' ) }
+						</Button>
+					) }
+				</div>
+			</div>
+		</Card>
+	);
+}
+
+export default function InvoicesList(): ReactElement {
+	const translate = useTranslate();
+	const invoices = useInvoicesQuery();
+
+	return (
+		<div className="invoices-list">
+			<Card compact className="invoices-list__header">
+				<div className="invoices-list__row">
+					<div>{ translate( 'Due Date' ) }</div>
+					<div>{ translate( 'Status' ) }</div>
+					<div>{ translate( 'Total' ) }</div>
+					<div></div>
```

Comment:
We could use the compact version here since it doesn't contain any text: `<div/>`